docs/production/email.md

```diff
@@ -12,7 +12,9 @@ email addresses and send notifications.
 2. Fill out the section of `/etc/zulip/settings.py` headed "Outgoing
    email (SMTP) settings".  This includes the hostname and typically
    the port to reach your SMTP provider, and the username to log into
-   it as.
+   it as. Ensure that your SMTP server allows emails originating from
+   the email address listed as `ZULIP_ADMINISTRATOR` in
+   `/etc/zulip/settings.py`.
```

Comment:
We probably should mention the full set of email senders Zulip uses here (or link to the troubleshooting block about it?).

Reply:
Pretty new to the Zulip code base. Realised this issue while setting up a server. Where would I find the full set of email senders? The [troubleshooting section](https://zulip.readthedocs.io/en/stable/production/email.html#advanced-troubleshooting) doesn't seem to have a list.

Reply:
Yeah, I'll just expand this commit to cover that :)